src/os/bluestore/BlueStore.cc

```diff
@@ -3973,7 +3973,7 @@ int BlueStore::_open_fm(bool create)
 	start += l + u;
       }
     }
-    db->submit_transaction_sync(t);
+    cct->_conf->bluestore_debug_omit_kv_commit ? 0 : db->submit_transaction_sync(t);
```

Comment:
may be we can let the submit_tx() get through, this part of the code is mostly on boot up time. Allocators and free list manager might need to store the info in DB.  If we want to measure any performance changes after bootup the debug flag will help. Similar in mkfs().